address/book.go

```diff
@@ -70,6 +70,20 @@ type QueryParams struct {
 	UnmanagedOnly bool
 }
 
+// AssetSyncer is an interface that allows the address.Book to look up asset
+// genesis and group information from both the local asset store and assets
+// known to universe servers in our federation.
+type AssetSyncer interface {
+	// FetchAssetInfo queries the universes in our federation for genesis
+	// and asset group information about the given asset ID.
+	FetchAssetInfo(ctx context.Context, assetID *asset.ID) error
+
+	// UpdateSyncConfig updates the sync config for the given asset so that
+	// we sync future issuance proofs.
+	UpdateSyncConfig(ctx context.Context, assetID *asset.ID,
```

Comment:
Can use `AssetGroup` here. 

I think we also only care about updating the sync config if the asset itself has a group.

Reply:
Yeah, I guess an ungrouped asset only ever needs to be synced once and therefore never needs a universe config option? For issuance proofs at least.